src/components/Modal/Modal.vue

```diff
@@ -33,24 +33,37 @@
 						{{ title }}
 					</div>
 					<div class="icons-menu">
+						<!-- Actions menu -->
 						<Actions class="header-actions">
 							<slot name="actions" />
 						</Actions>
-						<a v-if="hasNext && enableSlideshow" class="play-pause" @click="togglePlayPause">
+
+						<!-- Play-pause toggle -->
+						<button v-if="hasNext && enableSlideshow"
+							v-tooltip.auto="playPauseTitle"
```

Comment:
I haven't found this in the documentation, but I assume that `auto` is the default position. Hence, if it makes no difference, I would remove `.auto`.

Reply:
I'm not sure! :thinking: